test.sh

```diff
@@ -181,9 +188,8 @@ if [[ "$RUN" =~ "integration" ]] ; then
     echo "--- Recommend setting \$CERTBOT_PATH to  ---"
     echo "--- client repo with initialized virtualenv  ---"
     echo "------------------------------------------------"
-    run git clone -b v0.8.0 \
-      https://www.github.com/certbot/certbot.git \
-      $CERTBOT_PATH || exit 1
+    run git clone https://www.github.com/certbot/certbot.git $CERTBOT_PATH || exit 1
```

Comment:
Since we're now running 0.8.0, we should be compatible with the tests on master, and we can just check out master. Not the best long-term solution, but we can revert to checking out a tag that matches the certbot release next time there's an incompatible change to certbot's integration tests (it's rare).